fiftyone_pipeline_core/src/fiftyone_pipeline_core/pipelinebuilder.py

```diff
@@ -174,10 +174,25 @@ def build_from_configuration(self, config):
         for element in config["PipelineOptions"]["Elements"]:
             flow_element = importlib.import_module(element["elementPath"]).__getattribute__(element["elementName"])
             if("elementParameters" in element):
-                flow_element = flow_element(**element["elementParameters"])
+                arguments  = self._map_properties_names(
+                    {
+                        "verify_md5": "data_update_verify_md5",
+                        "data_file_update_base_url": "data_update_url"
+                    },
+                    **element["elementParameters"],
+                )
+
+                flow_element = flow_element(**arguments)
             else:
                 flow_element = flow_element()
             flow_elements.append(flow_element)
 
         return Pipeline(flow_elements)
 
+    def _map_properties_names(self, mappings, **arguments):
```

Comment:
1. `arguments` can be just a `dict` - you don't have to pack and unpack it with **.  
2. could you please add a small unit test for this method